test/unit/Slider.spec.js

```diff
@@ -0,0 +1,275 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import sinon from 'sinon';
+import keycode from 'keycode';
+import Slider from 'src/slider';
+
+describe('<Slider />', () => {
+  sinon.assert.expose(assert, {prefix: ''});
+
+  const getThumbElement = function(shallowWrapper) {
+    return shallowWrapper.children().at(2).children().at(0).children().at(2);
+  };
+
+  const getTrackContainer = function(shallowWrapper) {
+    return shallowWrapper.children().at(2);
+  };
+
+  it('renders slider and the hidden input', () => {
+    const wrapper = shallow(
+      <Slider name="slider" />
+    );
+
+    assert.ok(wrapper.find('input[type="hidden"]').length, 'should contain a hidden input');
+  });
+
+  it('renders slider with an initial value', () => {
+    const wrapper = shallow(
+      <Slider name="slider" value={0.5} />
+    );
+
+    assert.strictEqual(wrapper.state().value, 0.5, 'state.value should be equal the specified property');
+    assert.isAtLeast(wrapper.state().percent, wrapper.state().value);
+    assert.strictEqual(
+      wrapper.find('input[type="hidden"]').props().value,
+      wrapper.state().value,
+      'the input value should be equal state.value'
+    );
+  });
+
+  it('renders slider as a required element in a form', () => {
+    const wrapper = shallow(
+      <Slider name="slider" required={true} />
+    );
+
+    assert.isTrue(wrapper.find('input[type="hidden"]').props().required);
+  });
+
+  it('checks root node properties', () => {
+    const rootStyle = {
+      backgroundColor: 'red',
+    };
+    const wrapper = shallow(
+      <Slider name="slider" style={rootStyle} />
+    );
+
+    assert.deepEqual(wrapper.props().style, rootStyle, 'root element should have the style object');
+  });
+
+  it('checks slider initial state', () => {
+    const wrapper = shallow(
+      <Slider name="slider" />
+    );
+
+    assert.isFalse(wrapper.state().active, 'state.active should be initially false');
+    assert.isFalse(wrapper.state().dragging, 'state.dragging should be initially false');
+    assert.isFalse(wrapper.state().focused, 'state.focused should be initially false');
+    assert.isFalse(wrapper.state().hovered, 'state.hovered should be initially false');
+  });
+
+  it('checks drag start state', () => {
+    const handleDragStart = sinon.spy();
+    const wrapper = shallow(
+      <Slider name="slider" onDragStart={handleDragStart} />
+    );
+
+    wrapper.renderer._instance._instance._onDragStart();
```

Comment:
Hey @felipethome I had an idea for this that might make better use of the enzyme API.

If you find the inner node with the wrapper, you should be able to `shallow()` it and then use [`instance()`](https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/instance.md).

Let me know if it works! If not, no worries, I'll have a fiddle later.